spec/classes/puppet_server_spec.rb

```diff
@@ -353,7 +353,6 @@
         it do
           should contain_file(vardir)
             .with_ensure('directory')
-            .with_owner('root')
```

Comment:
If you add this back, does it still pass?

Reply:
good point, looks like it does!